src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/ndelius/Supervision.kt

```diff
@@ -6,15 +6,11 @@ import java.time.format.DateTimeFormatter
 
 data class Supervision(
   val mainOffence: MainOffence = MainOffence(),
+  val additionalOffences: List<AdditionalOffence> = listOf(AdditionalOffence()),
   val courtAppearances: List<CourtAppearance> = listOf(CourtAppearance()),
 ) {
-  fun toOffence(): Offence = Offence(
-    cjsCode = null,
-    hoCode = this.mainOffence?.code,
-    courtDates = this.courtAppearances.map { LocalDate.parse(it.date, DateTimeFormatter.ISO_OFFSET_DATE_TIME) }.filterNotNull(),
-    endDate = null,
-    startDate = null,
-    statuteCode = null,
-    description = this.mainOffence?.description,
-  )
+  fun toOffences(): List<Offence> {
+    val courtDates = courtAppearances.mapNotNull { LocalDate.parse(it.date, DateTimeFormatter.ISO_OFFSET_DATE_TIME) }
```

Comment:
Why doesn't this need a `this.` before it?

Reply:
Was an oversight, still works though. TIL: https://kotlinlang.org/docs/this-expressions.html#implicit-this
Will update for consistency.